logstash-core/spec/logstash/agent_spec.rb

```diff
@@ -9,6 +9,7 @@
 require_relative "../support/helpers"
 require_relative "../support/matchers"
 
+java_import org.logstash.Timestamp
```

Comment:
@jakelandis in the `master` version #7845 you have an empty line between this import and the `describe` line.
Maybe add that empty line here too, to keep the diff to `master` small, other than that also LGTM :)